game/scripts/npc/abilities/siltbreaker/lycan_boss_summon_wolves.txt

```diff
@@ -26,6 +26,15 @@
         //-------------------------------------------------------------------------------------------------------------
         "AbilityManaCost"               "20"
 
+        // Stats
```

Comment:
these need to be in the 
```
         "AbilitySpecial"
         {
             "01"
             {
.....
```

`GetSpecialValueFor` returns the AbilitySpecial value with that name at the current level.

Reply:
okay i'll fix that later ... roleplaying right now and i'll still have to do furion/phantom lancer custom tooltips